src/extension.ts

```diff
@@ -740,11 +754,15 @@ export class Extension implements RunHooks {
     return result.join('\n');
   }
 
-  private _testMessageFromText(text: string): vscodeTypes.TestMessage {
+  private _testMessageFromText(text: string, aiContext?: string): vscodeTypes.TestMessage {
     const markdownString = new this._vscode.MarkdownString();
     markdownString.isTrusted = true;
     markdownString.supportHtml = true;
     markdownString.appendMarkdown(ansi2html(this._abbreviateStack(text)));
+
+    if (aiContext)
+      markdownString.appendMarkdown(`\n`.repeat(10) + `## Context for AI\n${aiContext}`);
```

Comment:
let's make sure this isn't as in-your-face! <br> or details or small font-size, anything